src/OpenTelemetry.Exporter.NewRelic/NewRelicTraceExporter.cs

```diff
@@ -87,9 +87,12 @@ internal NewRelicTraceExporter(SpanDataSender spanDataSender, TelemetryConfigura
         /// <inheritdoc />
         public override ExportResult Export(in Batch<Activity> batch)
         {
+            // Prevent exporter's HTTP operations from being instrumented.
+            using var scope = SuppressInstrumentationScope.Begin();
+
             var nrSpanBatch = ToNewRelicSpanBatch(batch);
 
-            if (nrSpanBatch.Spans.Count == 0)
+            if (nrSpanBatch.Spans?.Count == 0)
```

Comment:
Can Spans be NULL?

Reply:
It can be null. This was actually a bug I introduced in a #96. The OpenTelemetry SDK interface for exporters changed. The `Batch<Activity>` is a readonly struct, so a check like `batch == null` or `batch == default` is not a thing. `ToNewRelicSpanBatch` will enumerate over the batch and may result in nrSpanBatch.Spans == null. We could consider solving this differently and ensure the Spans property is never null... might be nicer.